big_tests/tests/service_mongoose_system_metrics_SUITE.erl

```diff
@@ -291,6 +293,9 @@ transport_mechanisms_are_reported(_Config) ->
 outgoing_pools_are_reported(_Config) ->
     mongoose_helper:wait_until(fun outgoing_pools_are_reported/0, true).
 
+xmpp_stanzas_counts_are_reported(_Config) ->
+    mongoose_helper:wait_until(fun xmpp_stanzas_counts_are_reported/0, true).
```

Comment:
I think we could extend the test by connecting some users, sending some stanzas and checking if the reported value changed. Is is possible? Currently the test only check if the metric is reported, but it can be reported with wrong values. It'd be good to verify that.